setup.py

```diff
@@ -45,4 +49,5 @@ def run(self):
     long_description = f.read()
 
 
-setup(ext_modules=extensions, cmdclass={"build_ext": SimulatorBuild} if sys.platform == "win32" else {}),
+setup(ext_modules=extensions, cmdclass={"build_ext": SimulatorBuild})
+# setup(ext_modules=extensions, cmdclass={"build_ext": SimulatorBuild} if sys.platform in ["linux", "win32", "darwin"] else {}),
```

Comment:
Can you please remove commented out code?

Reply:
Done!